apps/frontend/src/renderer/components/linear/components/LinearTicketDetail.tsx

```
@@ -0,0 +1,235 @@
+            )}
+            {validation.taskProperties && (
+              <div>
+                <span className="font-medium">{t('linear:category')}:</span>{' '}
+                {validation.taskProperties.category}
+              </div>
+            )}
+          </div>
+          <button
+            onClick={() => setShowValidationModal(true)}
+            className="mt-3 text-sm text-green-700 dark:text-green-300 underline hover:no-underline"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)